mqtt/mqtt-broker/config/production.json

```diff
@@ -1,14 +1,11 @@
 {
```

Comment:
Let's move this file to `mqtt/contrib/edgehub/broker.json`
The folder `mqtt/contrib/edgehub/` might have more config files potentially (mqtt bridge) for instance.